CareKit/CareKit/Shared/Task/Controller/OCKTaskController.swift

```diff
@@ -444,6 +444,9 @@ open class OCKTaskController: ObservableObject {
         let detailViewController = OCKDetailViewController(showsCloseButton: true)
         detailViewController.detailView.titleLabel.text = task.title
         detailViewController.detailView.bodyLabel.text = task.instructions
+        if let asset = task.asset {
+            detailViewController.detailView.imageView.image = UIImage(named: asset)
+        }
```

Comment:
```suggestion
        detailViewController.detailView.imageView.image = task.asset.map(UIImage.init(named:))        
```

Reply:
This one-liner might be just a little bit cleaner. Check to make sure it compiles though!

Reply:
It looks like it returns `UImage??`.

Reply:
`detailViewController.detailView.imageView.image = task.asset.map(UIImage.init(named:)) ?? nil` would work. Let me know which of the options compared to below works best